charts/operator-wandb/charts/flat-run-fields-updater/templates/serviceaccount.yaml

```
@@ -0,0 +1,13 @@
+{{- if .Values.serviceAccount.create -}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "flat-run-fields-updater.serviceAccountName" . }}
+  labels:
+    {{- include "flat-run-fields-updater.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

To improve code readability, let's apply the same labeling and annotation patterns across the codebase.
```suggestion
  labels:
    {{- include "wandb.commonLabels" . | nindent 4 }}
    {{- include "flat-run-fields-updater.commonLabels" . | nindent 4 }}
    {{- include "flat-run-fields-updater.labels" . | nindent 4 }}
    {{- if .Values.serviceAccount.labels -}}
    {{-   toYaml .Values.serviceAccount.labels | nindent 4 }}
    {{- end }}
  annotations:
    {{- if .Values.serviceAccount.annotations -}}
    {{-   toYaml .Values.serviceAccount.annotations | nindent 4 }}
    {{- end }}
```